Checks/PWR037/benchmark/solution.c

```diff
@@ -0,0 +1,18 @@
+// PWR037: Potential precision loss in call to mathematical function
+
+#include <math.h>
```

Comment:
Ensure that `add_benchmark` links with `libm`. It seems this linkage might be missing!

Reply:
You may cherry-pick this commit https://github.com/codee-com/open-catalog/pull/58/commits/ec8a029315b48ac60d5f70abcf88e75f74bea222

Reply:
Thanks for the catch and the reference! I will cherry-pick that :)